src/librbd/cache/pwl/AbstractWriteLog.cc

```diff
@@ -1250,6 +1250,11 @@ void AbstractWriteLog<I>::complete_op_log_entries(GenericLogOperations &&ops,
     if (op->reserved_allocated()) {
       published_reserves++;
     }
+    {
```

Comment:
Why move this part line 1253- 1257 here?

Reply:
Need this since `m_dirty_log_entries` doesn't show up to date entries during internal_flush